Unable to build Fuzzel-1.7.0

Unable to build fuzzel at nixos-unstable

### Metadata
nixpkgs: github:nixos/nixpkgs/ce8cbe3c01fd8ee2de526ccd84bbf9b82397a510/

```console
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/s3zlm403hkqskr9mg0pnkd9ycaqzwzdd-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
meson flags: --buildtype=release         --libdir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/lib --libexecdir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/libexec         --bindir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/bin --sbindir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/sbin         --includedir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/include         --mandir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/share/man --infodir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/share/info         --localedir=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/share/locale         -Dauto_features=enabled         -Dwrap_mode=nodownload         --prefix=/nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0 -Denable-cairo=enabled -Dpng-backend=libpng -Dsvg-backend=librsvg 
[1mThe Meson build system[0m
Version: 0.60.3
Source dir: [1m/build/source[0m
Build dir: [1m/build/source/build[0m
Build type: [1mnative build[0m
Project name: [1mfuzzel[0m
Project version: [1m1.7.0[0m
C compiler for the host machine: [1mgcc[0m (gcc 10.3.0 "gcc (GCC) 10.3.0")
C linker for the host machine: [1mgcc[0m ld.bfd 2.35.2
Host machine cpu family: [1mx86_64[0m
Host machine cpu: [1mx86_64[0m
Checking for function "[1mmemfd_create[0m" : [1;32mYES[0m 
Compiler for C supports arguments -fmacro-prefix-map=/foo=: [1;32mYES[0m 
Library [1mm[0m found: [1;32mYES[0m
Run-time dependency [1mthreads[0m found: [1;32mYES[0m
Library [1mstdthreads[0m found: [1;31mNO[0m
Found pkg-config: [1m/nix/store/wgmpzzpgrgkigjlzyls03wv0132f6m39-pkg-config-wrapper-0.29.2/bin/pkg-config[0m (0.29.2)
Run-time dependency [1mfontconfig[0m found: [1;32mYES[0m [36m2.13.94[0m
Run-time dependency [1mpixman-1[0m found: [1;32mYES[0m [36m0.38.4[0m
Run-time dependency [1mcairo[0m found: [1;32mYES[0m [36m1.16.0[0m
Did not find CMake 'cmake'
Found CMake: [1;31mNO[0m
Run-time dependency [1mepoll-shim[0m found: [1;31mNO[0m (tried pkgconfig and cmake)
Run-time dependency [1mwayland-protocols[0m found: [1;32mYES[0m [36m1.25[0m
Run-time dependency [1mwayland-client[0m found: [1;32mYES[0m [36m1.20.0[0m
Run-time dependency [1mwayland-cursor[0m found: [1;32mYES[0m [36m1.20.0[0m
Run-time dependency [1mxkbcommon[0m found: [1;32mYES[0m [36m1.4.0[0m
Run-time dependency [1mtllist[0m found: [1;32mYES[0m [36m1.0.4[0m
Run-time dependency [1mfcft[0m found: [1;32mYES[0m [36m3.0.1[0m
Run-time dependency [1mlibpng[0m found: [1;32mYES[0m [36m1.6.37[0m
Run-time dependency [1mlibrsvg-2.0[0m found: [1;32mYES[0m [36m2.54.0[0m
Found pkg-config: [1m/nix/store/wgmpzzpgrgkigjlzyls03wv0132f6m39-pkg-config-wrapper-0.29.2/bin/pkg-config[0m (0.29.2)
Build-time dependency [1mwayland-scanner[0m found: [1;32mYES[0m [36m1.20.0[0m
Program [1m/nix/store/fmbfh6x95kf5x32rc00bsai0vg5yxr4h-wayland-1.20.0-bin/bin/wayland-scanner[0m found: [1;32mYES[0m (/nix/store/fmbfh6x95kf5x32rc00bsai0vg5yxr4h-wayland-1.20.0-bin/bin/wayland-scanner)
Program [1menv[0m found: [1;32mYES[0m (/nix/store/2w6011wwg5v6qxz182r4d0mmr3i4rxm1-coreutils-9.0/bin/env)
Program [1msh[0m found: [1;32mYES[0m (/nix/store/dndi916j6yxzfzzj2sma2llhrlwahq06-bash-5.1-p16/bin/sh)
Build-time dependency [1mscdoc[0m found: [1;32mYES[0m [36m1.11.2[0m
Program [1m/nix/store/f05j6mw39afvjwcwbvgfklrxmy83fcbb-scdoc-1.11.2/bin/scdoc[0m found: [1;32mYES[0m (/nix/store/f05j6mw39afvjwcwbvgfklrxmy83fcbb-scdoc-1.11.2/bin/scdoc)
Build targets in project: [1m9[0m

fuzzel [36m1.7.0[0m

    Cairo        : [1;32mYES[0m
    PNG icons    : libpng
    SVG icons    : librsvg

  [1mUser defined options[0m
    auto_features: enabled
    bindir       : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/bin
    buildtype    : release
    includedir   : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/include
    infodir      : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/share/info
    libdir       : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/lib
    libexecdir   : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/libexec
    localedir    : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/share/locale
    mandir       : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/share/man
    prefix       : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0
    sbindir      : /nix/store/hmdr428jf0jgxfs9138c5fwi6y1ca7n9-fuzzel-1.7.0/sbin
    wrap_mode    : nodownload
    enable-cairo : enabled
    png-backend  : libpng
    svg-backend  : librsvg

Found ninja-1.10.2 at /nix/store/r59p34gpbf2h3x0vn9v2vggkdwk7vaql-ninja-1.10.2/bin/ninja
meson: enabled parallel building
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j12 -l12
[1/26] Generating external_wlr_layer_shell_unstable_v1_xml-private-code with a custom command
[2/26] Generating __nix_store_hxjaidnr8nhjm3mdz50a5zl5c2m1qp2v_wayland_protocols_1_25_share_wayland_protocols_unstable_xdg_output_xdg_output_unstable_v1_xml-client-header with a custom command
[3/26] Generating external_wlr_layer_shell_unstable_v1_xml-client-header with a custom command
[4/26] Generating __nix_store_hxjaidnr8nhjm3mdz50a5zl5c2m1qp2v_wayland_protocols_1_25_share_wayland_protocols_unstable_xdg_output_xdg_output_unstable_v1_xml-private-code with a custom command
[5/26] Generating __nix_store_hxjaidnr8nhjm3mdz50a5zl5c2m1qp2v_wayland_protocols_1_25_share_wayland_protocols_stable_xdg_shell_xdg_shell_xml-private-code with a custom command
[6/26] Generating generate_version with a custom command
[7/26] Generating __nix_store_hxjaidnr8nhjm3mdz50a5zl5c2m1qp2v_wayland_protocols_1_25_share_wayland_protocols_stable_xdg_shell_xdg_shell_xml-client-header with a custom command
[8/26] Compiling C object fuzzel.p/meson-generated_.._wlr-layer-shell-unstable-v1.c.o
[9/26] Compiling C object fuzzel.p/meson-generated_.._xdg-shell.c.o
[10/26] Compiling C object fuzzel.p/meson-generated_.._xdg-output-unstable-v1.c.o
[11/26] Compiling C object fuzzel.p/char32.c.o
[12/26] Compiling C object fuzzel.p/log.c.o
[13/26] Compiling C object fuzzel.p/fdm.c.o
[14/26] Compiling C object fuzzel.p/png.c.o
[15/26] Compiling C object fuzzel.p/prompt.c.o
[16/26] Compiling C object fuzzel.p/dmenu.c.o
FAILED: fuzzel.p/dmenu.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/dmenu.c.o -MF fuzzel.p/dmenu.c.o.d -o fuzzel.p/dmenu.c.o -c ../dmenu.c
In file included from ../application.h:7,
                 from ../dmenu.h:5,
                 from ../dmenu.c:1:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
[17/26] Generating doc/fuzzel.1 with a custom command (wrapped by meson to capture output)
[18/26] Compiling C object fuzzel.p/shm.c.o
[19/26] Compiling C object fuzzel.p/icon.c.o
FAILED: fuzzel.p/icon.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/icon.c.o -MF fuzzel.p/icon.c.o.d -o fuzzel.p/icon.c.o -c ../icon.c
In file included from ../application.h:7,
                 from ../icon.h:5,
                 from ../icon.c:1:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
[20/26] Compiling C object fuzzel.p/application.c.o
FAILED: fuzzel.p/application.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/application.c.o -MF fuzzel.p/application.c.o.d -o fuzzel.p/application.c.o -c ../application.c
In file included from ../application.h:7,
                 from ../application.c:1:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
[21/26] Compiling C object fuzzel.p/match.c.o
FAILED: fuzzel.p/match.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/match.c.o -MF fuzzel.p/match.c.o.d -o fuzzel.p/match.c.o -c ../match.c
In file included from ../application.h:7,
                 from ../match.h:6,
                 from ../match.c:1:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
[22/26] Compiling C object fuzzel.p/main.c.o
FAILED: fuzzel.p/main.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/main.c.o -MF fuzzel.p/main.c.o.d -o fuzzel.p/main.c.o -c ../main.c
In file included from ../application.h:7,
                 from ../main.c:25:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
[23/26] Compiling C object fuzzel.p/render.c.o
FAILED: fuzzel.p/render.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/render.c.o -MF fuzzel.p/render.c.o.d -o fuzzel.p/render.c.o -c ../render.c
In file included from ../application.h:7,
                 from ../match.h:6,
                 from ../render.h:9,
                 from ../render.c:1:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
[24/26] Compiling C object fuzzel.p/xdg.c.o
FAILED: fuzzel.p/xdg.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/xdg.c.o -MF fuzzel.p/xdg.c.o.d -o fuzzel.p/xdg.c.o -c ../xdg.c
In file included from ../application.h:7,
                 from ../xdg.h:3,
                 from ../xdg.c:1:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
[25/26] Compiling C object fuzzel.p/wayland.c.o
FAILED: fuzzel.p/wayland.c.o 
gcc -Ifuzzel.p -I. -I.. -I/nix/store/0kc611kqhjcqiljhmh49z9n21bvxm1nq-pixman-0.38.4/include/pixman-1 -I/nix/store/d6wg6wf5qb0cdlnhmz9w7ladgcklf6bj-fontconfig-2.13.94-dev/include -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include/freetype2 -I/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/include/cairo -I/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/include -I/nix/store/yfckbbm9qf5ql7ghvygqmw4d6idib76f-zlib-1.2.11-dev/include -I/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/include/libpng16 -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include -I/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/include/glib-2.0 -I/nix/store/miqmafvlv5al1df7fvhdlr13fl2zn6jh-glib-2.72.0/lib/glib-2.0/include -I/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0 -I/nix/store/m7bf8ba0sz25b57jlr0dcyj24w9npbxw-gdk-pixbuf-2.42.8-dev/include/gdk-pixbuf-2.0 -I/nix/store/1skm7nv8fy8ygjcvk8p4ag6v9g4mq8my-wayland-1.20.0-dev/include -I/nix/store/0c0j85r7xb5cp5l8xl1bmd9bj47mpsq0-libxkbcommon-1.4.0-dev/include -I/nix/store/6w1jic3f5r2pknililgx8fskm776jkrq-tllist-1.0.5/include -I/nix/store/x2syxpr3nx04ilgdxnbn9icxk51h8cnw-fcft-3.0.1/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_GNU_SOURCE -DMEMFD_CREATE -fmacro-prefix-map=../= -DFUZZEL_ENABLE_CAIRO=1 -DFUZZEL_ENABLE_PNG_LIBPNG=1 -DFUZZEL_ENABLE_SVG_LIBRSVG=1 -pthread -MD -MQ fuzzel.p/wayland.c.o -MF fuzzel.p/wayland.c.o.d -o fuzzel.p/wayland.c.o -c ../wayland.c
In file included from ../application.h:7,
                 from ../dmenu.h:5,
                 from ../wayland.h:5,
                 from ../wayland.c:1:
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:66: error: "/*" within comment [-Werror=comment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |                                                                   
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:85: error: "/*" within comment [-Werror=comment]
/nix/store/9gx8wp6f7a83i19aqlwzdkl14psn3wiw-librsvg-2.54.0-dev/include/librsvg-2.0/librsvg/rsvg.h:125:92: error: "/*" within comment [-Werror=comment]
cc1: all warnings being treated as errors
ninja: build stopped: subcommand failed.
```

--- a/pkgs/applications/misc/fuzzel/default.nix
+++ b/pkgs/applications/misc/fuzzel/default.nix
@@ -60,6 +60,8 @@ stdenv.mkDerivation rec {
     "-Dsvg-backend=${withSVGBackend}"
   ];
 
+  CFLAGS = "-Wno-error=comment"; # https://gitlab.gnome.org/GNOME/librsvg/-/issues/856
+
   meta = with lib; {
     description = "Wayland-native application launcher, similar to rofi’s drun mode";
     homepage = "https://codeberg.org/dnkl/fuzzel";

